composer.json

```diff
@@ -14,8 +14,8 @@
         "ezsystems/ezpublish-kernel": "^6.0|^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.35|^5.7",
-        "matthiasnoback/symfony-dependency-injection-test": "~1.0"
+        "phpunit/phpunit": "^4.8.35|^5.7|^6.0",
```

Comment:
Do we still need PHPUnit 4?

Reply:
we have PHP 5.5

Reply:
maybe it is time to drop support for PHP 5.5 and 5.6, that should be something to 3.0 I suppose ?

Reply:
Doesn't have to be. Bumping dependency requirements is not a breaking change: http://doctrine-project.org/2017/07/25/php-7.1-requirement-and-composer.html